db/migrate/20170905111153_create_user_agents.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUserAgents < ActiveRecord::Migration
+  def change
+    create_table :user_agents do |t|
+      t.string :user_agent,           default: "", :limit => 1000
```

Comment:
I don't think this makes sense to be not nullable with an empty default. Either there is a user agent string or there isn't.